### PR TITLE
feat: verbose Failed to fetch error

### LINKED ISSUE #4560: `TypeError: failed to fetch` is vague and unpleasant to look at when a request fails
### Content & configuration

Example Swagger/OpenAPI definition:
```yaml
openapi: 3.0.0
servers: 
- url: http://kyles.hockey:1234/this/wont/work
info:
  title: test
  version: test
paths:
  '/':
    get:
      responses:
        '200':
          description: Successful response
```

### Current behavior

image

`TypeError: failed to fetch` appears, giving me no clues as to what went wrong - a refused connection, in this case.

### To reproduce...

1. Expand the operation in the provided example document
2. Click `Execute` to fire an ill-fated request

### Desired behavior

A styled UI element that appears _instead_ of the regular live response display UI would more clearly indicate that no response was received at all, and displaying the browser's error reason would give me actionable information without having to open my browser console.

### Additional context or thoughts

cc #3403.

### PR DESCRIPTION
**Pull request by mathis-m**
Signed-off-by: mathis-m <mathis.michel@outlook.de>

### Description

make TypeError: Failed to fetch more verbose.
According to specification https://fetch.spec.whatwg.org/ it is not possible to distinguish between network errors: 
> A network error is a response whose status is always 0, status message is always the empty byte sequence, header list is always empty, and body is always null.

It is only viewable via developer tools e.g. console.

In order to make the ui information more verbose I added the following text to clarify the current response state:
```js
"A network error occurred. This could be a CORS issue or a dropped internet connection. It is not possible for us to know."
```
### Motivation and Context

Fixes #4560

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by tim-lai** [comment]
@mathis-m What happens right now if it's an invalid url provided as compared to a no-network available? Is it a different `err.message`?

**Comment by mathis-m** [comment]
@tim-lai the error we can see in developer console is not inferable via javascript this is a security feature.
So we can choose to enhance the error message in case the error is the generic error "Failed to fetch."
There is no way to tell why this error occured, besides having a look at the developer console.
In order to enhance the error message we could do something like this:
image

**Comment by tim-lai** [comment]
@mathis-m Ok, thanks for the update. However, let's remove references to the console. To follow linting best practices, I also think we should remove or comment out the `console.error` entirely.

File: src/core/plugins/spec/actions.js [hunk 1]
```diff
@@ -477,6 +477,10 @@ export const executeRequest = (req) =>
     .catch(
       err => {
         console.error(err)
```
Comment (tim-lai) [inline src/core/plugins/spec/actions.js:479]:
```suggestion
        // console.error(err)
```

File: src/core/plugins/spec/actions.js [hunk 2]
```diff
@@ -476,7 +476,11 @@ export const executeRequest = (req) =>
     } )
     .catch(
       err => {
-        console.error(err)
+        // console.error(err)
+        if(err.message === "Failed to fetch") {
+          err.name = ""
+          err.message = "**Failed to fetch.**  \n**See console** for more information.  \n**Possible Reasons:** \n  - CORS \n  - Network Failure \n  - URL scheme must be \"http\" or \"https\" for CORS request."
```
Comment (tim-lai) [inline src/core/plugins/spec/actions.js:482]:
```suggestion
          err.message = "**Failed to fetch.**  \n**Possible Reasons:** \n  - CORS \n  - Network Failure \n  - URL scheme must be \"http\" or \"https\" for CORS request."
```

**Comment by mathis-m** [comment]
@tim-lai Thanks for the cleanup. I did not have the time to work on that!
